docs/Backend_tutorials/features/easy_queries.md

```diff
@@ -0,0 +1,215 @@
+# How to create new EasyQuery
+
+This section provide basic information what needs to be implemented for new working easy query.
+
+## Easy Query
+Easy queries source codes are located in directory `app/easy_queries`. Easy query must be inherited from `EasyQuery` class.
+New easy query name is name of model with suffix `EasyQuery`. For example `MyModelEasyQuery` for model `MyModel`. 
+Every easy query need to be registered to be able use it within dashboards. 
+
+Although it is possible to namespace easy queries, for now `EasyGraphql::Extensions::EasyQuery` extension does not support it.
```

Comment:
I recommend do not split it into 2 groups => Just keep "namespace are not supported" 🙂